spec/features/user_sees_average_salary_spec.rb

```diff
@@ -0,0 +1,16 @@
+#As a user,  When I visit a specific job page,
+  #I also see an average salary for all jobs within each specific tag listed.
+
+require 'rails_helper'
+RSpec.describe "Average Salary" do
+  it "a user sees average salary for all jobs for each specific tag" do
+
+    company = Company.create(name: "Turning")
+    job = company.jobs.create(title: "Ag", description: "work with plants", level_of_interest: 98, city: "Denver", salary: 10)
+    job2 = company.jobs.create(title: "Space", description: "work with planets", level_of_interest: 95, city: "Denver", salary: 20)
+    job3 = company.jobs.create(title: "Tech", description: "work with tech", level_of_interest: 90, city: "Denver", salary: 30)
+    job4 = company.jobs.create(title: "Gov", description: "work with rocks", level_of_interest: 91, city: "Denver", salary: 40)
+
+
```

Comment:
Remember that our tests are the thing that we know we want to be true about our code. Despite the issue with creating a join table, there's still every reason to write our tests to expect the creation and relation of Tags for Jobs. Hopefully then doing that leads to a solution for said join table.